src/controllers/ratings.controller.js

```diff
@@ -0,0 +1,73 @@
+import db from '../database/models';
```

Comment:
Good job @codexempire , you might also want to consider renaming this controller to `rating.controller.js` without the `s` to maintain the our consistency in naming and also in for the className?